packages/ses/docs/lockdown.md

```diff
@@ -366,12 +367,65 @@ the container to exit explicitly, and we highly recommend setting
 - `'none'`: do not install traps for uncaught exceptions. Errors are likely to
   appear as `{}` when they are reported by the default trap.
 
+## `evalTaming` Options
+
+This option only affects the start compartment!
+
+To disallows eval in the explicit compartments, replace the constructors in the compartment.
+
+```js
+const c = new Compartment()
+c.globalThis.eval = c.globalThis.Function = function() {
```

Comment:
wait, what about hidden intrinsics? 🤔

AsyncFunction, GeneratorFunction, ...?

Reply:
For a compartment, its own evaluator is not reachable unless the reference to itself is leaked.

Does it mean I should do some change on this PR?